src/goto-programs/goto_convert_side_effect.cpp

```diff
@@ -440,7 +438,7 @@ void goto_convertt::remove_cpp_new(
   decl.add_source_location()=new_symbol.location;
   convert_decl(decl, dest, ID_cpp);
 
-  call=code_assignt(new_symbol.symbol_expr(), expr);
+  auto call=code_assignt(new_symbol.symbol_expr(), expr);
```

Comment:
I believe this should be `const code_assignt call(new_symbol.symbol_expr(), expr);`